org.adempiere.ui.zk/WEB-INF/src/web/theme/default/css/fragment/find-window.css.dsp

```diff
@@ -13,3 +13,24 @@
 .find-window-advanced .img-btn, .find-window-advanced .img-btn.btn-ok.z-button, .find-window-advanced .img-btn.btn-cancel.z-button {
 	margin: 0 2px;
 }
+ 
+.find-window .z-window-content{
+	padding-bottom: 0px;	
+}
+
+.find-window .status{
+    height: 20px;
+}
+
+.find-window .statusbar{
+    height: 22px;
```

Comment:
The amount of white space is a subjective matter and it is not appropriate to make this change as part of a bug fix of other issues.